media/css/firefox/home/protocol.scss

```diff
@@ -87,8 +74,15 @@ main .mzp-l-content {
         background-position: top left 50vw;
         background-repeat: no-repeat;
         display: flex;
-        min-height: 560px;
+        min-height: 610px;
         text-align: left;
+        box-sizing: border-box;
+
+        .test-var-e & {
+            background-image: url('/media/img/firefox/home/var-e-product.svg'), url('/media/img/firefox/home/var-e-bg.svg');
+            background-position: left calc(50% + 348px) top 72px, top -110px left 48vw;
+            background-size: 593px auto, 1055px auto;
```

Comment:
Nit: `background-size()` mixin?

Reply:
Given our support matrix I think we can retire that mixin. Should I use it here for consistancy's sake?

Reply:
Your call. The only real reason I still use it is because it doesn't degrade gracefully.